src/settingsKeyMapping.js

```diff
@@ -560,6 +561,18 @@ const settingsKeyMapping = {
         label: i18n.t('Enable password expiry alerts'),
```

Comment:
Two suggestions here:

- This label uses the term `alert`, whereas we are writing `reminder` in the . I'd suggest using a single term, `reminder` as it hints at the purpose (reminding users to change password).
- We can include _the user_ here to make it clear that reminders will be sent to users, not to the admins. 

Suggestion based on above:
`Send reminders to users before their password expires`